curvefs/src/metaserver/metastore.cpp

```diff
@@ -110,6 +189,10 @@ bool MetaStoreImpl::Clear() {
         PartitionCleanManager::GetInstance().Remove(it->first);
     }
     partitionMap_.clear();
+
+    // auto ret = kvStorage_->Close();
```

Comment:
Delete.There will also be new partitions created.

May be need `MetaStoreImpl::Destory() `to close storage before metaserver exit.

Reply:
fix, `MetaStoreImpl::Destory()` is a good idea, but currently we don't need it.